packages/phone-number-privacy/combiner/package.json

```diff
@@ -23,17 +23,19 @@
     "test:e2e": "jest test/end-to-end --verbose",
     "test:e2e:staging": "ODIS_COMBINER_SERVICE_URL=https://us-central1-celo-phone-number-privacy-stg.cloudfunctions.net yarn test:e2e",
     "test:e2e:alfajores": "ODIS_COMBINER_SERVICE_URL=https://us-central1-celo-phone-number-privacy.cloudfunctions.net yarn test:e2e",
+    "test:e2e:mainnet": "ODIS_COMBINER_SERVICE_URL=https://us-central1-celo-pgpnp-mainnet.cloudfunctions.net yarn test:e2e",
     "db:migrate": "NODE_ENV=dev FIREBASE_CONFIG=./firebase.json ts-node ./scripts/run-migrations.ts",
     "db:migrate:staging": "GCLOUD_PROJECT=celo-phone-number-privacy-stg yarn db:migrate",
     "db:migrate:alfajores": "GCLOUD_PROJECT=celo-phone-number-privacy yarn db:migrate",
     "db:migrate:mainnet": "GCLOUD_PROJECT=celo-pgpnp-mainnet yarn db:migrate",
     "db:migrate:make": "knex --migrations-directory ./migrations migrate:make -x ts"
   },
   "dependencies": {
-    "@celo/contractkit": "2.0.1-dev",
-    "@celo/identity": "2.0.1-dev",
-    "@celo/phone-number-privacy-common": "1.0.39",
-    "@celo/utils": "2.0.1-dev",
+    "@celo/contractkit": "2.0.0",
+    "@celo/identity": "2.0.0",
+    "@celo/phone-number-privacy-common": "1.0.41",
+    "@celo/phone-utils": "2.0.0",
+    "@celo/utils": "2.0.0",
```

Comment:
Do we want to change the local dependencies on `master` to non `-dev` versions? We could leave this change on the release branch instead.